src/NLog.Web.AspNetCore/LayoutRenderers/AspNetRequestBodySynchronousIOAllowedLayoutRenderer.cs

```diff
@@ -0,0 +1,28 @@
+﻿#if ASP_NET_CORE3
+using NLog.LayoutRenderers;
+using NLog.Web.Internal;
+using System.Text;
+using Microsoft.AspNetCore.Http.Features;
+
+namespace NLog.Web.LayoutRenderers
+{
+    /// <summary>
+    /// Whether synchronous IO is allowed for the Body
+    /// Uses IHttpBodyControlFeature
+    /// </summary>
+    /// <remarks>
+    /// <code>${aspnet-request-body-synchronous-io-allowed}</code>
+    /// </remarks>
+    [LayoutRenderer("aspnet-request-body-synchronous-io-allowed")]
+    public class AspNetRequestBodySynchronousIOAllowedLayoutRenderer : AspNetLayoutRendererBase
```

Comment:
When is this relevant? Believe Sync-IO detectuib was a temporary work-around from NetCoreAp2.1 to NetCoreApp3.1

Reply:
Oh we can just remove that one then.  Was not aware of that.

Reply:
Removed